### PR TITLE
Added hotkey for grouping layers (Ctrl+G). Grid toggle hotkey changed to #.

### PR DESCRIPTION
**Pull request by rock16**
It's is easier to group selected layers with hotkeys. #2525 
I added an accelerator (`<Control>g`) in `synfig/synfig-studio/src/gui/app.cpp`  which is mapped to the layer action for layer encapsulation, Users will be able to group selected layers using the hotkey `Ctrl + G`.

`Ctrl + G` was initially used for grid togling which i replaced with the pound sign "#". so one is now able to toggle the grid with `#` and group Selected layers with `Ctrl + G`.

### CODE REVIEW

File: synfig-studio/src/gui/app.cpp [hunk 1]
```diff
@@ -1339,6 +1339,7 @@ App::get_default_accel_map()
 		{"Delete",                  "<Actions>/action_group_layer_action_manager/action-LayerRemove"},
 		{"<Control>parenleft" ,     "<Actions>/canvasview/decrease-low-res-pixel-size"},
 		{"<Control>parenright" ,    "<Actions>/canvasview/increase-low-res-pixel-size"},
+		{"<Control>g",				"<Actions>/action_group_layer_action_manager/action-LayerEncapsulate"},
```
Comment (rodolforg) [inline synfig-studio/src/gui/app.cpp:1342]: Please use spaces for alignment between accelerator and action path strings.

**Review by rodolforg — CHANGES_REQUESTED** [review]

**Comment by ice0** [comment]
@rock16 

Hi, Rock!

1. Please use `<Primary>` key instead of `<Control>`.

> a new facility is provided in Gtk+ (as of this writing it is in Git for Gtk+-2.24, and released in Gtk+-3.2.0) to use the <Primary> descriptor in place of <Control> for accelerators and bindings. This will map the accelerator to Command on MacOS and to Control for anything else

2. Is it possible to add Ungroup action?

**Comment by rock16** [comment]
@rodolforg 

> Please use spaces for alignment between accelerator and action path strings.

I will fix that now.

> Is it possible to add Ungroup action?

Should I submit a different pull request for the Ungroup action?

**Comment by rodolforg** [comment]
> Is it possible to add Ungroup action?

Ctrl+Z ? XD

Well, a new synfigapp::Action have to be created for it, @ice0

**Comment by ice0** [comment]
> > Is it possible to add Ungroup action?
> 
> Ctrl+Z ? XD
> 
> Well, a new synfigapp::Action have to be created for it, @ice0

Agree. `Ctrl+Z` will be enough.

> Should I submit a different pull request for the Ungroup action?

No. I suggested this because I read in the comments about Group / Ungroup, but now I understand that it does not make sense.

**Review by rodolforg — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**